CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 # Changelog
 
 ## V 0.5.0
-- Renamed all `refresh_token` methods to access Token to be consistent
```

Comment:
I removed this line because I don't think it's relevant for someone wanted to understand what changed publicly. This type of comment is probably better for a commit than the changelog.